kernel/declarations.ml

```diff
@@ -151,9 +151,9 @@ type wf_paths = recarg Rtree.t
 
 (**
 {v
-   Inductive I1 (params) : U1 := c11 : T11 | ... | c1p1 : T1p1
+   Inductive I₁ (params) : U₁ := c₁₁ : T₁₁ | ... | c₁p₁ : T₁p₁
```

Comment:
No sure I like unicode subscripts, they get annoying when you want to use a subscript that's not in unicode

Reply:
This is not fully convincing, I agree.